doc/man7/provider-keymgmt.pod

```diff
@@ -278,8 +291,40 @@ dimensions handled in the rest of the same provider.
 The value should be the number of security bits of the given key.
 Bits of security is defined in SP800-57.
 
+=item "ecdh-cofactor-mode" (B<OSSL_PKEY_PARAM_ECDH_COFACTOR_MODE>) <integer>
+
+The value should be either 1 or 0, to respectively enforce or disable the ECDH
+cofactor mode for the given key.
+
 =back
 
+=head1 RETURN VALUES
+
+OP_keymgmt_new() should return a valid reference to the newly created provider
+side key object, or NULL on failure.
+
+OP_keymgmt_import(), OP_keymgmt_export(), OP_keymgmt_get_params() and
+OP_keymgmt_set_params() should return 1 for success or 0 on error.
+
+OP_keymgmt_validate() should return 1 on successful validation, or 0 on
+failure.
+
+OP_keymgmt_has() should return 1 if all the selected data subsets are contained
+in the given I<keydata> or 0 otherwise.
+
+OP_keymgmt_query_operation_name() should return a pointer to a string matching
+the requested operation, or NULL if the same name used to fetch the keymgmt
+applies.
+
+OP_keyexch_init(), OP_keyexch_set_peer(), OP_keyexch_derive(),
+OP_keyexch_set_params(), and OP_keyexch_get_params() should return 1 for success
+or 0 on error.
```

Comment:
This bit is in the wrong file...

Reply:
Fixed, it was an artifact of copy&paste!